File: Oops Notes/NOTES FOR STUDIES/OOPS EXAMPLE/PSITStaticMethod_Class/PSITStaticMethod_Class/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace PSITStaticMethod_Class
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        //GD
        static class Test
        {
            ////public void print()
            ////{
            ////    MessageBox.Show("From print");
            ////}
            public static void display()
            {
                MessageBox.Show("From display");
            }
        }
        private void button1_Click(object sender, EventArgs e)
        {
            //Test t = new Test();
            //t.print();
            Test.display();
        }
    }
}

File: Oops Notes/NOTES FOR STUDIES/Abdul rub/VB/WindowsApplication1/WindowsApplication1/Form1.vb
Imports System.Data
Imports System.Data.SqlClient
Public Class Form1
    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim cnn As New SqlConnection(" Integrated Security=SSPI;Persist Security Info=False;Initial Catalog=student3;Data Source=vikas\SQLEXPRESS")
        Dim Sql As String
        Sql = "select loginname from userinfo where loginname='" & txtloginname.Text & "' and passward='" & txtpassword.Text & "'"
        Dim da As New SqlDataAdapter(Sql, cnn)
        Dim ds As New DataSet
        da.Fill(ds, "a")
        If ds.Tables("a").Rows.Count > 0 Then
            g_loginname = txtloginname.Text
            Form2.Show()
            Me.Hide()
        Else
            MessageBox.Show("Invalid login name or Password")

        End If

    End Sub

    Private Sub TextBox1_TextChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txtloginname.TextChanged

    End Sub

    Private Sub txtloginname_KeyPress(ByVal sender As System.Object, ByVal e As System.Windows.Forms.KeyPressEventArgs) Handles txtloginname.KeyPress
        If Asc(e.KeyChar) = Keys.Enter Then
            txtpassword.Focus()
        End If
    End Sub

    Private Sub txtpassword_KeyPress(ByVal sender As System.Object, ByVal e As System.Windows.Forms.KeyPressEventArgs) Handles txtpassword.KeyPress
        If Asc(e.KeyChar) = Keys.Enter Then
            txtloginname.Focus()
        End If
    End Sub

    Private Sub txtloginname_Enter(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txtloginname.Enter
        txtloginname.SelectAll()
    End Sub

    Private Sub txtpassword_Enter(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txtpassword.Enter
        txtpassword.SelectAll()
    End Sub

    Private Sub txtpassword_TextChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txtpassword.TextChanged

    End Sub
End Class
